trio/_core/tests/test_run.py

```diff
@@ -1709,59 +1693,25 @@ async def main():
 
         _core.run(main)
 
-    class Deferred:
-        "Just kidding"
+    for bad_call in bad_call_run, bad_call_spawn:
 
-    with ignore_coroutine_never_awaited_warnings():
-        for bad_call in bad_call_run, bad_call_spawn:
+        async def f():  # pragma: no cover
+            pass
 
-            async def f():  # pragma: no cover
-                pass
+        with ignore_coroutine_never_awaited_warnings():
+            with pytest.raises(TypeError, match="expecting an async function"):
 
```

Comment:
No need for an extra blank line after `with pytest.raises`.

Reply:
You marked this as resolved but didn't change anything in your PR. Do you feel like the extra blank lines here are important for readability in a way that's related to the changes you're making here? It seems like you've added a number of them in this diff after block introducers (for, with, if, etc). That's not consistent with the style in the rest of the codebase.